tests/data/README.md

```diff
@@ -0,0 +1,5 @@
+# apdb.sqlite3
+
+This file was copied from `lsst-devl01:/home/swnelson/profiling-workspace/temp_apdb_a1.db`
```

Comment:
Do we expect this path to be long-lived?

(Also, I suspect you can just say `lsst-devl`, and leave the `01` as an implementation detail...)

Reply:
I don't expect that to be long lived. I wanted some kind of provenance here, to show where it came from. I'd rather have a programmatic way to generate a small APDB with a small handful of sources for the future, since we'll have to update it a few times to reflect schema changes.